### DISCUSSION TITLE
datetime input

### DISCUSSION THREAD

**Discussion by hyusetiawan** [django-commons/django-unicorn#562] [Q&A]
I have the following in the html template:
`<input type="date" placeholder="Publish" unicorn:model="publish" />`

and in the view, I have a datetime model, like so:
```
class NewsView(UnicornView):
    keyword = ""
    source = ""
    publish: datetime = datetime.now()
```

what's the best way to connect these two?

**Comment by adamghill** [comment]
I'm not sure I understand what you mean by "connect these two"? What are you looking to do?
